execution-service/src/main/java/org/ihtsdo/rvf/execution/service/WhitelistService.java

```diff
@@ -0,0 +1,30 @@
+package org.ihtsdo.rvf.execution.service;
+
+import org.ihtsdo.rvf.execution.service.whitelist.AcceptanceGatewayClient;
+import org.ihtsdo.rvf.execution.service.whitelist.AcceptanceGatewayClientFactory;
+import org.ihtsdo.rvf.execution.service.whitelist.WhitelistItem;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class WhitelistService {
+
+    @Autowired
+    private AcceptanceGatewayClientFactory factory;
+
+    public boolean isWhitelistDisabled() {
+        return factory.isWhitelistDisabled();
+    }
+
```

Comment:
Could you just add a comment here to clarify what is happening please?
Something like:
```
// Check which component validation failures are on the AAG whitelist.
```